fastlane/lanes/build.rb

```diff
@@ -136,31 +168,54 @@
     sentry_upload_dsym(
       auth_token: get_required_env('SENTRY_AUTH_TOKEN'),
       org_slug: SENTRY_ORG_SLUG,
-      project_slug: 'jetpack-ios',
+      project_slug: SENTRY_PROJECT_SLUG_JETPACK,
       dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
     )
   end
 
-  # Builds the app and uploads it to App Center
+  # Builds the "WordPress Internal" app and uploads it to App Center
   #
   # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
+  # @option [Boolean] skip_prechecks (default: false) If true, don't run the ios_build_prechecks and ios_build_preflight
   #
   # @called_by CI
-  # @calls build_and_upload_internal
   #
   desc 'Builds and uploads for distribution to App Center'
   lane :build_and_upload_app_center do |options|
-    ios_build_prechecks(
-      skip_confirm: options[:skip_confirm],
-      internal: true,
-      external: true
+    ios_build_prechecks(skip_confirm: options[:skip_confirm], internal: true) unless options[:skip_prechecks]
+    ios_build_preflight unless options[:skip_prechecks]
+
+    sentry_check_cli_installed
+
+    internal_code_signing
+
+    gym(
+      scheme: 'WordPress Internal',
+      workspace: WORKSPACE_PATH,
+      export_method: 'enterprise',
+      clean: true,
+      output_directory: BUILD_PRODUCTS_PATH,
+      output_name: 'WordPress Internal',
+      derived_data_path: DERIVED_DATA_PATH,
+      export_team_id: get_required_env('INT_EXPORT_TEAM_ID'),
+      export_options: { method: 'enterprise' }
```

Comment:
Nitpick on argument's position, since I saw you addressed [the same suggestion](https://github.com/wordpress-mobile/WordPress-iOS/pull/18140#discussion_r832798229) in #18140:

```suggestion
      clean: true,
      output_directory: BUILD_PRODUCTS_PATH,
      output_name: 'WordPress Internal',
      derived_data_path: DERIVED_DATA_PATH,
      export_team_id: get_required_env('INT_EXPORT_TEAM_ID'),
      export_method: 'enterprise',
      export_options: { method: 'enterprise' }
```